packages/ui5-task-i18ncheck/readme.md

```diff
@@ -0,0 +1,56 @@
+# UI5 task for checking missing translations in i18n properties used in XML views
```

Comment:
can you make the filename uppercase please -> README.md?

Reply:
Sure, 
`git mv -f readme.md README.md`
By the way, the zipper task has the same lowercase `readme.md` I haven't changed that to keep pull request focused.